prepare_data/src/test/groovy/org/orbisgis/osm/FormattingForAbstractModelTests.groovy

```diff
@@ -289,6 +290,7 @@ class FormattingForAbstractModelTests {
         new OSMGISLayersTests().prepareVegetTest()
         def h2GIS = H2GIS.open('./target/h2db')
```

Comment:
use another name for the db. osmdb or something like that.
It'd be easier for debug purpose

Reply:
Done